packages/frontend/src/pages/scaling-activity/props/getActivityView.ts

```diff
@@ -41,3 +47,28 @@ export function getActivityViewEntry(
     transactionsWeeklyCount,
   }
 }
+
+function getEthereumActivityViewEntry(
+  ethereum: Ethereum,
```

Comment:
Why do we need to pass this all the way from the config package? I think having `name: 'Ethereum'` and `slug: 'ethereum'` hardcoded will be much cleaner